CHANGELOG.md

```diff
@@ -30,6 +34,7 @@
 
 This release contains contributions from (in alphabetical order):
 Lillian M. A. Frederiksen
+Austin Huang
```

Comment:
Should be in alphabetical order.

Reply:
It is in alphabetical order.... unless I'm missing something?

Reply:
The middle names are a bit tricky!